tests/conftest.py

```diff
@@ -972,11 +982,30 @@ def milvus_server_uri():
         logger.info("Started Milvus at: %s", uri)
 
         yield uri
-    except Exception as exec_inf:
-        logger.error("Error in starting Milvus server: %s", exec_inf)
-        raise
-    finally:
-        try:
-            default_server.stop()
-        except Exception as exec_inf:
-            logger.error("Error in stopping Milvus server: %s", exec_inf)
+
+
+@pytest.fixture(scope="function", name="milvus_service")
+def milvus_service_fixture(milvus_server_uri: str):
+    # This fixture is scoped to the function level since the WriteToVectorDBStage will close the connection on'
+    # pipeline completion
+    from morpheus.service.milvus_vector_db_service import MilvusVectorDBService
+    service = MilvusVectorDBService(uri=milvus_server_uri)
+    yield service
```

Comment:
This seems like it should be something scoped to the milvus tests. The conf tests should be free of objects we are testing

Reply:
This fixture is used by both `tests/test_milvus_vector_db_service.py` and by `tests/test_milvus_write_to_vector_db_stage_pipe.py`. 

As far as I'm aware the only way to allow them to share the fixture is to either place them here, or move the two milvus test to their own sub-directory.